OpenRA.Mods.Common/Traits/SupportPowers/SupportPowerManager.cs

```diff
@@ -231,10 +231,8 @@ public virtual void Target()
 				return;
 
 			var power = Instances.FirstOrDefault(i => !i.IsTraitPaused);
-			if (power == null)
-				return;
 
-			power.SelectTarget(power.Self, Key, Manager);
+			power?.SelectTarget(power.Self, Key, Manager);
```

Comment:
`Instances.FirstOrDefault(i => !i.IsTraitPaused)?.SelectTarget(power.Self, Key, Manager);`?

Reply:
Wont work because we are using `power.self` in the call.